test/e2e/apps/cronjob.go

```diff
@@ -412,17 +411,33 @@ var _ = SIGDescribe("CronJob", func() {
 		framework.ExpectEqual(string(createdCronJob.UID), statusUID, fmt.Sprintf("createdCronJob.UID: %v expected to match statusUID: %v ", createdCronJob.UID, statusUID))
 
 		// CronJob resource delete operations
-		ginkgo.By("deleting a collection")
-		expectFinalizer := func(cj *batchv1beta1.CronJob, msg string) {
+		expectFinalizer := func(cj *batchv1.CronJob, msg string) {
 			framework.ExpectNotEqual(cj.DeletionTimestamp, nil, fmt.Sprintf("expected deletionTimestamp, got nil on step: %q, cronjob: %+v", msg, cj))
 			framework.ExpectEqual(len(cj.Finalizers) > 0, true, fmt.Sprintf("expected finalizers on cronjob, got none on step: %q, cronjob: %+v", msg, cj))
 		}
+
+		ginkgo.By("deleting")
+		cjTemplate.Name = "for-removal"
+		forRemovalCronJob, err := cjClient.Create(context.TODO(), cjTemplate, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+		err = cjClient.Delete(context.TODO(), forRemovalCronJob.Name, metav1.DeleteOptions{})
+		framework.ExpectNoError(err)
+		cjs, err = cjClient.List(context.TODO(), metav1.ListOptions{LabelSelector: "special-label=" + f.UniqueName})
+		framework.ExpectNoError(err)
+		// Should have <= 2 items since some cronjobs might not have been deleted yet due to finalizers
+		framework.ExpectEqual(len(cjs.Items) <= 2, true, "filtered list should be <= 2")
+		// Validate finalizers
+		for _, cj := range cjs.Items {
+			expectFinalizer(&cj, "deleting cronjob collection")
```

Comment:
this is just `"deleting cronjob"` right? (will be confused with the one from like 443 otherwise?)

Reply:
Yeah, lemme update that quickly right away.